### DISCUSSION TITLE
Adjust seemingly inappropriate unreliability values for 3D Cinema, Flying Saucers

### DISCUSSION THREAD

**Discussion by reticulatingsplines** [OpenRCT2/OpenRCT2#15877] [Feature requests]
See pull request #15875 . Sorry for not posting a discussion first, tried to go through the guidelines for proposing changes, my bad. To repeat here:

The initial unreliability factors for a couple rides seem inappropriate. The 3D Cinema's unreliability_factor is set at 21; it's tied with the Motion Simulator, and only the Enterprise (22), Roto-Drop (24), Reverse Freefall Coaster (25), LIM-launched coaster (25), and Air-powered vertical coaster (28) are higher. Based on the stats, it seems like it shouldn't really be the same as the motion simulator - it's probably just meant to be a movie theater where you wear 3D glasses. IDK if anybody tinkered with these values over time, but perhaps if so, the digits got swapped? I think 12 makes sense, if not even still too high. The Miniature Railroad is 11, the Car Ride, Mini Helicopters, River Rafts, and Ghost Train are 12, and the Junior Coaster and Dinghy Slide are 13.

There is still actually one ride with a much higher value - the Flying Saucers, at 32. I think this is probably based on the history of the Flying Saucers ride at Disneyland in California, which opened in the 60s and was apparently notoriously hard to maintain - see https://en.wikipedia.org/wiki/Flying_Saucers_(attraction). Still, regular Dodgems comes at half of that, 16, and it's rainproof to boot, which already makes Flying Saucers feel like a somewhat pointless alternative. I proposed 17 just to make it a tad more unreliable than Dodgems because it's rain-exposed.

**Comment by pizza2004** [comment]
I think the 3D Cinema in this game is based on older technology where you'd have to often have two projectors and so stuff like the two projects becoming unsynced gets added to other stuff like the lens going out of focus.

Imagine a breakdown as just being something the projectionist would normally deal with and it makes sense, but I kinda agree that it's weird to have a full on mechanic go there for every such issue, since you'd imagine someone would be employed in the building to deal with the smaller issues.
